packages/fsml/fsml.0.2a/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/fsml"
+doc: "http://jserot.github.io/fsml"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "2.6"}
+  "menhir" {>= "20200624"}
+  "ocaml" {>= "4.10" & < "4.11"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jserot/fsml.git"
+
+url {
+  src: "https://github.com/jserot/fsml/archive/0.2a.tar.gz"
+  checksum: "md5=d66b77322b4808f98575daac8853ed51"
```

Comment:
It is still failing. I get `961e37e807f6c8722f7d7f4d29fcb8a7` as `md5sum` from the provided url. Can you check that it is the correct one?
```suggestion
  checksum: "md5=961e37e807f6c8722f7d7f4d29fcb8a7"
```

Reply:
Done